### PR TITLE
add TGI compat feature for rollingbatch

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Add TGI compatibility support

### CODE REVIEW

File: engines/python/setup/djl_python/tests/test_test_model.py [hunk 1]
```diff
@@ -54,7 +55,8 @@ def test_with_env(self):
         envs = {
             "OPTION_MODEL_ID": "NousResearch/Nous-Hermes-Llama2-13b",
             "SERVING_LOAD_MODELS": "test::MPI=/opt/ml/model",
-            "OPTION_ROLLING_BATCH": "auto"
+            "OPTION_ROLLING_BATCH": "auto",
+            "TGI_COMPAT": "true"
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tests/test_test_model.py:59]: Can we use SERVING_TGI_COMPAT?

Comment (lanking520) [reply engines/python/setup/djl_python/tests/test_test_model.py:59]: Why SERVING?

Comment (frankfliu) [reply engines/python/setup/djl_python/tests/test_test_model.py:59]: This should be a global server configuration and can be configure in `config.properties`

**Comment by sindhuvahinis** [comment]
If the env name is finalized, every thing else, looks good to me.

File: engines/python/setup/djl_python/rolling_batch/rolling_batch.py [hunk 2]
```diff
@@ -445,6 +451,8 @@ def __init__(self, **kwargs):
         self.waiting_steps = kwargs.get("waiting_steps", None)
         self.current_step = 0
         self.default_output_formatter = kwargs.get("output_formatter", None)
+        global TGI_COMPAT
+        TGI_COMPAT = kwargs.get("tgi_compat", "false").lower() == 'true'
```
Comment (frankfliu) [inline engines/python/setup/djl_python/rolling_batch/rolling_batch.py:455]:
```suggestion
        self.tgi_compat = kwargs.get("tgi_compat", "false").lower() == 'true'
```

Comment (lanking520) [reply engines/python/setup/djl_python/rolling_batch/rolling_batch.py:455]: There is no way to pass this value back to the serializer. Unless we make them to be part of the Request class

**Review by frankfliu — APPROVED** [review]

**Pull request merged**